test/unit/config/index.js

```diff
@@ -174,6 +174,73 @@ describe('Config', () => {
 
                 expect(() => new Config('./docpub.config')).to.throw(/string/);
             });
+
+            describe('renderer', () => {
+                it('should not throw if `renderer` is not provided', () => {
+                    stubConfig_({
+                        path: './docpub.config',
+                        config: {
+                            username: 'default_username',
+                            token: 'default_token',
+                            url: 'default_url'
+                        }
+                    });
+
+                    expect(() => new Config('./docpub.config')).to.not.throw();
+                });
+
+                it('should throw if `highlight` is not a boolean', () => {
+                    stubConfig_({
+                        path: './docpub.config',
+                        config: {
+                            username: 'default_username',
+                            token: 'default_token',
+                            url: 'default_url',
+                            renderer: {
+                                highlight: 'test'
+                            }
+                        }
+                    });
+
+                    expect(() => new Config('./docpub.config')).to.throw(/boolean/);
+                });
+            });
+        });
+
+        describe('defaults', () => {
```

Comment:
Not sure that we need specific `describe` for defaults. It should be easier to read tests grouped by option.